### PR TITLE
Add profile for otter-browser

### PR DESCRIPTION
**Pull request by kortewegdevries**
Request: # 1139

### CODE REVIEW

File: etc/profile-m-z/otter-browser.profile [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+# Persistent global definitions
+include globals.local
+
+?BROWSER_ALLOW_DRM: ignore noexec ${HOME}
+
+noblacklist ${HOME}/.pki
+noblacklist ${HOME}/.local/share/pki
+noblacklist ${HOME}/.cache/Otter
+noblacklist ${HOME}/.config/otter
+
```
Comment (Fred-Barclay) [inline etc/profile-m-z/otter-browser.profile:15]: alphabetise

File: etc/profile-m-z/otter-browser.profile [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+mkdir ${HOME}/.pki
+mkdir ${HOME}/.cache/Otter
+mkdir ${HOME}/.config/otter
+mkdir ${HOME}/.local/share/pki
+whitelist ${DOWNLOADS}
+whitelist ${HOME}/.pki
+whitelist ${HOME}/.cache/Otter
+whitelist ${HOME}/.config/otter
+whitelist ${HOME}/.local/share/pki
```
Comment (Fred-Barclay) [inline etc/profile-m-z/otter-browser.profile:31]: also alphabetise

File: etc/profile-m-z/otter-browser.profile [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+nonewprivs
+noroot
+notv
+?BROWSER_DISABLE_U2F: nou2f
+protocol unix,inet,inet6,netlink
+seccomp !chroot
+shell none
+
+disable-mnt
+private-cache
```
Comment (Fred-Barclay) [inline etc/profile-m-z/otter-browser.profile:52]: is a private-bin possible?

Comment (kortewegdevries) [reply etc/profile-m-z/otter-browser.profile:52]: `private-bin` will break right-clicking Downloads and opening files among other things. Opera,Falkon,Firefox Chromium-common, all have none or commented. Do you want me to add the basic stuff like `otter,sh,bash,which`?

File: etc/profile-m-z/otter-browser.profile (hunk 1 above)
Comment (kortewegdevries) [reply etc/profile-m-z/otter-browser.profile:15]: Thanks!

File: etc/profile-m-z/otter-browser.profile (hunk 3 above)
Comment (rusty-snake) [reply etc/profile-m-z/otter-browser.profile:52]: > will break right-clicking Downloads and opening files

I don't think we support this for other browsers

> Opera,Falkon,Firefox Chromium-common

The reason is that the most distros ship custom/own shell-scripts to start firefox.

File: etc/profile-m-z/otter-browser.profile [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+noblacklist ${HOME}/.local/share/pki
+noblacklist ${HOME}/.cache/Otter
+noblacklist ${HOME}/.config/otter
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
```
Comment (rusty-snake) [inline etc/profile-m-z/otter-browser.profile:21]: `include disable-xdg.inc`?

File: etc/profile-m-z/otter-browser.profile (hunk 3 above)
Comment (kortewegdevries) [reply etc/profile-m-z/otter-browser.profile:52]: I'll add the above `private-bin`.

File: etc/profile-m-z/otter-browser.profile (hunk 4 above)
Comment (kortewegdevries) [reply etc/profile-m-z/otter-browser.profile:21]: `sort.py` keeps copying each line twice!

**Comment by rusty-snake** [comment]
Merged, Thanks.

**Pull request merged**